code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -87,19 +87,18 @@
 		return
 	if((HUSK in target.mutations) || (!ishuman(target)))
 		to_chat(user, "<span class='warning'>Our sting appears ineffective against its DNA.</span>")
-		return 0
+		return FALSE
+	if(selected_dna)
```

Comment:
I didn't look at this close enough; this shouldn't be under `can_sting` this should be below

```
if(!selected_dna)
     return
``` 

under `Click`